RubberduckTests/Inspections/ObjectVariableNotSetInspectionTests.cs

```diff
@@ -304,9 +304,9 @@ End Enum
 
         // This is a corner case similar to #4037. Previously, Collection's default member was not being generated correctly in
         // when it was loaded by the COM collector (_Collection is missing the default interface flag). After picking up that member
-        // this test fails because it resolves as attempting to assign 'New Colletion' to `Test.DefaultMember`.
+        // this test fails because it resolves as attempting to assign 'New Collection' to `Test.DefaultMember`.
         [Test]
-        [Ignore("Broken by COM collector fix. See comment on test.")]
+        //[Ignore("Broken by COM collector fix. See comment on test.")]
```

Comment:
If it's working, do we really need the comment (both of them) anymore?

Reply:
Oh, I missed go remove the ignore line entirely. Will do that later.